app/frontend/src/components/forms/manage/ManageVersions.vue

```diff
@@ -163,12 +263,28 @@ export default {
       return this.drafts && this.drafts.length > 0;
     },
     versionList() {
-      return this.form ? this.form.versions : [];
+      if(this.hasDraft){
```

Comment:
..this was my idea to merge the draft object into the array of versions.
basically formatting like the version object but with 'draft: true' property.